sycl/test/esimd/dpas.cpp

```diff
@@ -331,12 +331,12 @@ SYCL_ESIMD_FUNCTION SYCL_EXTERNAL void xmx_func() {
     // ------------ DPASW FP16: WITH THE ACCUMULATOR OPERAND -------------------
     R_f = xmx::dpasw<8, 1, float>(C_f, B_hf, A_hf);
     zoo(R_f);
-    // CHECK: call <8 x float> @llvm.genx.dpasw.v8f32.v64i32.v4i32(<8 x float> {{[^,]+}}, <64 x i32> {{[^,]+}}, <4 x i32> {{[^,]+}}, i32 17304074)
+    // CHECK: call <8 x float> @llvm.genx.dpasw.v8f32.v64i32.v4i32(<8 x float> {{[^,]+}}, <64 x i32> {{[^,]+}}, <4 x i32> {{[^,]+}}, i32 17303817)
 
     // ------------ DPASW FP16: WITHOUT ACC OPERAND ----------------------------
     R_f = xmx::dpasw<8, 1, float>(B_hf, A_hf);
     zoo(R_f);
-    // CHECK: call <8 x float> @llvm.genx.dpasw.nosrc0.v8f32.v64i32.v4i32(<64 x i32> {{[^,]+}}, <4 x i32> {{[^,]+}}, i32 17304074)
+    // CHECK: call <8 x float> @llvm.genx.dpasw.nosrc0.v8f32.v64i32.v4i32(<64 x i32> {{[^,]+}}, <4 x i32> {{[^,]+}}, i32 17303817)
```

Comment:
Can you tell why this new check is correct?
That last number is created this way:
  constexpr int Info = (RepeatCount << 24) + (SystolicDepth << 16) +
                       ((int)APrecision << 8) + (int)BPrecision;
17303817 = 0x01.08.09.09. I.e. APrecision = 9, while fp16=10

What actually happened here is the copy-paste error at the lines 328,329 above. The idea of this section was to check sycl::half, while those B_hf and A_hf are declared as simd<bfloat16,*> instead of simd<sycl::half,*>

Reply:
For this call `BPrecision` is `9`.